scanner/src/test/java/ca/on/oicr/gsi/runscanner/scanner/processor/RevioPacBioProcessorTest.java

```diff
@@ -0,0 +1,47 @@
+package ca.on.oicr.gsi.runscanner.scanner.processor;
+
+import ca.on.oicr.gsi.runscanner.dto.NotificationDto;
+import ca.on.oicr.gsi.runscanner.dto.PacBioNotificationDto;
+import ca.on.oicr.gsi.runscanner.dto.type.Platform;
+import ca.on.oicr.gsi.runscanner.scanner.processor.RunProcessor.Builder;
+import java.io.File;
+import java.io.IOException;
+import java.util.TimeZone;
+import org.junit.Assert;
+
+public class RevioPacBioProcessorTest<T> extends AbstractProcessorTest<PacBioNotificationDto> {
+  private final RevioPacBioProcessor instance =
+      new RevioPacBioProcessor(new Builder(Platform.PACBIO, "unittest", null));
+  private NotificationDto afterProcessing;
+
+  public RevioPacBioProcessorTest() {
+    super(PacBioNotificationDto.class);
+  }
+
+  @Override
+  protected NotificationDto process(File directory) throws IOException {
+    afterProcessing = instance.process(directory, TimeZone.getTimeZone("America/Toronto"));
+    return afterProcessing;
+    // return instance.process(directory, TimeZone.getTimeZone("America/Toronto"));
+  }
+
+  @Override
+  public void beforeComparison(PacBioNotificationDto reference, PacBioNotificationDto result) {
+    // Check if reference run name matches new run directory name
+    if (afterProcessing.getRunAlias().equals(reference.getRunAlias())) {
+      // Assert that a start time (any) was detected
+      Assert.assertNotNull(afterProcessing.getStartDate());
+
+      // Need to modify reference.json because for runs that have just started, the only
+      // information they have are the run name and start time using file creation. For tests, we
+      // cannot use file creation time as these aren't stored in git.
+      // Set the start date to match the reference json
+      afterProcessing.setStartDate(reference.getStartDate());
```

Comment:
Also hard-code the date you're setting to make sure it's doing what you think it is.